.env.example

```
@@ -5,6 +5,7 @@ COMPARTMENT_BUILDKIT_RESOURCES='{"requests":{"cpu":"250m","memory":"512Mi"},"lim
 COMPARTMENT_BUILD_NAMESPACE=compartment-build
 COMPARTMENT_BUILD_RUNNER_RESOURCES='{"requests":{"cpu":"100m","memory":"256Mi"},"limits":{"cpu":"250m","memory":"1Gi"}}'
 COMPARTMENT_BUILD_TIMEOUT_MS=1800000
+COMPARTMENT_DEPLOYMENT_INFRASTRUCTURE_TIMEOUT_MS=600000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COMPARTMENT_DEPLOYMENT_INFRASTRUCTURE_TIMEOUT_MS key should go before the COMPARTMENT_ENV key

(UnorderedKey)